doc/source/Linux.rst

```diff
@@ -0,0 +1,190 @@
+For plugin requests, Please create an issue with description of the plugin.
+
+.. code-block:: shell-session
+
+    $ vol3 --help | grep -i linux. | head -n 5
+        banners.Banners     Attempts to identify potential linux banners in an
+        linux.bash.Bash     Recovers bash command history from memory.
+        linux.check_afinfo.Check_afinfo
+        linux.check_creds.Check_creds
+        linux.check_idt.Check_idt
+
+
+Using plugins
+-------------
+
+The following is the syntax to run volatility tool.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f <path to memory image> plugin_name plugin_option
+
+
+Example
+-------
+
+Example 1
+~~~~~~~~~
+
+In this example we will be using memory dump from Insomni'hack teaser 2020 CTF. Challenge name Getdents, you can find the memory dump
+in the link `here <https://www.mediafire.com/file/5n0khasfir0wpsv/memory_challenge.zip/file>`_ . We will limit the discussion to memory forensics with volatility3 and not extend to other parts of the challenges.
+I'd like to say thanks to `stuxnet <https://github.com/stuxnet999/>`_ for providing this memory dump and `writeup <https://stuxnet999.github.io/insomnihack/2020/09/17/Insomihack-getdents.html>`_.
+
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f memory.vmem banners
+        
+        Volatility 3 Framework 2.0.3
+
+        Progress:  100.00               PDB scanning finished
+        Offset  Banner
+
+        0x141c1390      Linux version 4.15.0-42-generic (buildd@lgw01-amd64-023) (gcc version 7.3.0 (Ubuntu 7.3.0-16ubuntu3)) #45-Ubuntu SMP Thu Nov 15 19:32:57 UTC 2018 (Ubuntu 4.15.0-42.45-generic 4.15.18)
+        0x63a00160      Linux version 4.15.0-72-generic (buildd@lcy01-amd64-026) (gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)) #81-Ubuntu SMP Tue Nov 26 12:20:02 UTC 2019 (Ubuntu 4.15.0-72.81-generic 4.15.18)
+        0x6455c4d4      Linux version 4.15.0-72-generic (buildd@lcy01-amd64-026) (gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)) #81-Ubuntu SMP Tue Nov 26 12:20:02 UTC 2019 (Ubuntu 4.15.0-72.81-generic 4.15.18)
+        0x6e1e055f      Linux version 4.15.0-72-generic (buildd@lcy01-amd64-026) (gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)) #81-Ubuntu SMP Tue Nov 26 12:20:02 UTC 2019 (Ubuntu 4.15.0-72.81-generic 4.15.18)
+        0x7fde0010      Linux version 4.15.0-72-generic (buildd@lcy01-amd64-026) (gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)) #81-Ubuntu SMP Tue Nov 26 12:20:02 UTC 2019 (Ubuntu 4.15.0-72.81-generic 4.15.18)
+
+
+This above command helps us to find the memory dump's kernel version and the distribution version. Now using the above banner we can search for ISF file from the ISF server.
+If you do not find the ISF file then, please follow the instructions on :ref:`Linux:Procedure to create symbol tables for linux`. After that place the ISF file under ``volatility3/symbols/linux`` directory.
```

Comment:
As pointed out above, this may change.  There's currently work going on to change the requirement of where the ISF files (see #630).